scripts/create_pkg.sh

```diff
@@ -1,18 +1,44 @@
 #!/bin/bash
```

Comment:
Again the changes should be reverted, we don't need this for local lambda

Reply:
resolved